src/main/java/seedu/contax/storage/StorageManager.java

```diff
@@ -71,8 +76,35 @@ public void saveAddressBook(ReadOnlyAddressBook addressBook) throws IOException
 
     @Override
     public void saveAddressBook(ReadOnlyAddressBook addressBook, Path filePath) throws IOException {
-        logger.fine("Attempting to write to data file: " + filePath);
+        logger.fine("Attempting to write to address book data file: " + filePath);
         addressBookStorage.saveAddressBook(addressBook, filePath);
     }
 
```

Comment:
minor nit, can follow the existing format and add a comment before these set of methods to split it
```suggestion
        // ================ Schedule methods ==============================
```

Reply:
Fixed in commit a0ce0f6.